Source/WebKit/Platform/IPC/HandleMessage.h

```diff
@@ -95,6 +95,16 @@ void logMessageImpl(const Connection& connection, MessageName messageName, void*
 #endif
 }
 
+#if ENABLE(IPC_TRACE)
+inline void IPCTrace(MessageName messageName, Decoder& decoder)
+{
+    TextStream stream(TextStream::LineMode::SingleLine, { }, loggingContainerSizeLimit);
+    size_t length = decoder.span().size_bytes();
+    stream << description(messageName) << "," << length;
+    RELEASE_LOG(IPCTrace, "%s", stream.release().utf8().data());
```

Comment:
This doesn't work on any platform that uses `ENABLE_LOGD_BLOCKING_IN_WEBCONTENT`, which includes all embedded Apple platforms, since logging itself is implemented with IPC on those platforms.

Reply:
Hm it looks like that means we are getting an extra log line per IPC in the WebContent process, but it sounds like that is what you want.

A nit here is that I would definitely not use TextStream here. The logging macro takes a printf format string, just use it directly.

I also wonder if this should be declared as inline code in the header or if it should be moved to HandleMessage.cpp. I don't think we want to bloat code size for something like this.

Reply:
> I also wonder if this should be declared as inline code in the header or if it should be moved to HandleMessage.cpp. I don't think we want to bloat code size for something like this.

I think I will look at moving it to HandleMessage.cpp and exporting a symbol.